app/views/CorporationTaxEnquiriesView.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import views.html._
+
+@this(
+    main_template: MainTemplate,
+    appConfig: AppConfig)
+
+@()(implicit request: Request[_], messages: play.api.i18n.Messages)
+
+@main_template(title = messages("global.nav.title")) {
+
+@components.back_link()
+
+    <h1 class="header-margin-top">@messages("corporation.tax.enquiries.title")</h1>
+
+    <div class="govuk-inset-text">
+        <p>@messages("global.opening.times.title")</p>
+        <p>@messages("global.opening.times.8_30to4")</p>
+        <p>@messages("global.opening.times.advisers.p")</p>
```

Comment:
missing "Closed weekends and bank holidays." as per prototype ?
https://dep-prototype.herokuapp.com/fiveLinks/corporate

Reply:
done